src/server/api/schema.js

```diff
@@ -1255,16 +1255,62 @@ const rootResolvers = {
         contactsFilter,
         utc
       },
-      { user }
+      { user },
+      info
     ) => {
+      const includeTags = graphqlInfo => {
```

Comment:
Can this be moved somewhere and explain what it's for.

These root endpoints should be short and clear. Maybe `info` (maybe call it requestInfo or graphQLInfo) should just be passed to getConversations() and then includeTags can be a function in that library (and outside the function) rather than here?